firewalld/files/zone.xml

```diff
@@ -91,7 +107,10 @@
     <protocol value="{{ rule.protocol }}"/>
   {%- endif %}    
   {%- if 'icmp_block' in rule %}
-    <icmp_block name="{{ rule.icmp_block }}"/>
+    <icmp-block name="{{ rule.icmp_block }}"/>
+  {%- endif %}    
+  {%- if 'icmp_type' in rule %}
+    <icmp-type name="{{ rule.icmp_type }}"/>
```

Comment:
[Likewise](https://github.com/saltstack-formulas/firewalld-formula/pull/27/files#r272809459).

Reply:
should be fixed